src/WingetCreateCLI/FieldValidation.cs

```diff
@@ -59,6 +59,11 @@ public static class FieldValidation
 
                     // If the original type of the field is not a string, validate as a List<string>
                     property = items;
+                }
+                else if (type == typeof(long))

```

Comment:
>else if (type == typeof(long)) [](http://example.com/codeflow?start=16&length=30)

It seems a littleoff to put a single scalr type in the same level as a list type

Reply:
Modified this so that I'm checking for scalar type at the same level as list type, then handling the scalar types individually as needed